pkgs/applications/misc/lilyterm/default.nix

```diff
@@ -1,17 +1,39 @@
-{ stdenv, fetchurl
+{ stdenv, fetchurl, fetchFromGitHub
 , pkgconfig
 , autoconf, automake, intltool, gettext
-, gtk, vte }:
+, gtk, vte
 
```

Comment:
To separate the the “manual” arguments from the “automatic” ones (i.e. included by callPackage by default).